index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Project3</title>
+  </head>
+  <body>
+    <header>
+      <h1>
+        my conference header
+      </h1>
+    </header>
+    <nav>
+      <ul>
+        <li><a href="#About">About</a></li>
+        <li><a href="#Schedule">Schedule</a></li>
+        <li><a href="#Speakers">Speakers</a></li>
+        <li><a href="#Register">Register</a></li>
+      </ul>
+    </nav>
+    <main>
+      <p id="About">
+        <!--Site content. Conference Descriptions, images, etc.-->
+        <!-- About -->
+        This is a conference for every designers who uses types and letterings.
+        This year's conference will be hosted online on February 28, 2021.
+      </p>
+      <p id="Schedule">
+        Conference will be on Feb.28, 2021 9:00 A.M.- 7:00 P.M. 9:00 A.M.:
+        Introduction 10:00 A.M.- 7:00 P.M.: Typographic/Lettering Design Fair
+        10:30 A.M.: Hand Lettering workshop 1:00 P.M.: Typeface as a design tool
+        2:30 P.M.: Visual Typography
+      </p>
+      <p id="Speakers">
```

Comment:
This is a good example of the note I made above -- you probably want a separate `p` for each of the speakers. And possibly also some additional styling/emphasis on their names (which could be handled via a `strong` element or a `span`).